### DISCUSSION TITLE
Fail to load Phantom Camera addon 🥹

### DISCUSSION THREAD

**Discussion by tnhsy** [ramokz/phantom-camera#230] [General]
I'm fairly new to Godot and game development. I downloaded an addon from GitHub, but I'm running into some errors when I try to add it to my Godot project. This prevents me from opening the example scene.
My Godot is 4.2.1
<img width="882" alt="截屏2024-03-10 19 05 36" src="https://github.com/ramokz/phantom-camera/assets/13975515/cca32001-7b68-4061-bdd0-6b61f74d10f4">

**Comment by tnhsy** [comment]
Also... I'm wondering if later we will have a video tutorial that shows how to add and configure the Phantom Camera 2D.  I think a visual guide would be very helpful for me at this stage.

**Reply by ramokz** [reply]
A tutorial would be great, for sure! However, I myself can't make any as the addon is still in active development, so anything I make today would be out-of-date in a few months, which takes time away from improving the addon, unfortunately.

There are a few videos on YouTube already, including one that goes into how he [uses the `PCam2D` in his scene](youtube.com/watch?v=85K0e2Y0U3Q).  It is in Spanish, but the Auto Closed Caption does a pretty good job at translating it, if that's of any help.

**Comment by ramokz** [comment]
That's odd. Haven't seen those errors before.

Are you using either Mobile or Compatibility render mode, or is it set to Forward+ (top-right corner of the Godot editor window)?

**Reply by tnhsy** [reply]
I've solved this problem by switching the project to "Forward+" or "Mobile", I was using "Compatibility" before! Thanks a lot!!!!

**Reply by ramokz** [reply]
You're welcome!

There is an [existing bug report](https://github.com/ramokz/phantom-camera/issues/177) that's tracking this.

But I think this is a general Godot bug, rather than a addon specific one, so there's unfortunately no ETA for when it gets resolved if you need to use the Compatibility render.
